src/main/java/emissary/core/MobileAgent.java

```diff
@@ -760,7 +765,22 @@ protected void logAgentCompletion(final IBaseDataObject payloadArg) {
         final Logger dest = (isProbe == null) ? logger : probeLogger;
 
         if (dest.isInfoEnabled()) {
-            dest.info(PayloadUtil.getPayloadDisplayString(payloadArg));
+            // grab reduced-transform-itinerary-list if it exists and add lines to reducedTransformItineraryList
+            List<String> reducedTransformItineraryList;
+            try (Stream<String> lines =
+                    Files.lines(Paths.get(System.getenv("PROJECT_BASE") + "/../src/test/resources/reduced-transform-itinerary-list.csv"))) {
```

Comment:
Another thing is I think this is loading the configuration each time for every object. We should probably just load it once at startup.